src/web/templates/history.html

```diff
@@ -99,6 +99,7 @@ <h1 style="font-size: 26px; font-weight: bold; font-family: 'Roboto', sans-serif
       {% set year = request.form.get('year', '2425') %}
       {% set semester = request.form.get('semester', '1') %}
       {% set type = request.form.get('type', 'ug') %}
+      {% set start_round_num = 0 if year in ['2122', '2223', '2324'] else 1 %}
```

Comment:
Is it possible to import the constants `PRE_2425_ROUNDS`, `POST_2425_ROUNDS` and `PRE_2425_YEARS` via `context_processor()` in `src/web/app.py`?

Then, we can reuse the logic by saying something like `set round_nums = PRE_2425_ROUNDS if year in PRE_2425_YEARS else POST_2425_ROUNDS` so there is less code duplication. On line 111, the `round_num` can then be directly derived from `round_nums`.